storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -985,6 +985,37 @@ private static boolean isTopologyActiveOrActivating(IStormClusterState state, St
         return isTopologyActive(state, topoName) || state.activeStorms().contains(topoName);
     }
 
+    /**
+     * Returns the topologyId of the topology using this blob.
+     * @param state the cluster state
+     * @param topoCache the topology cache
+     * @param key the blob key
+     * @return null or id
+     */
+    private static String topologyUsingThisBlob(IStormClusterState state, TopoCache topoCache, String key) {
+        for (String topologyId : state.activeStorms()) {
+            Map<String, Object> topoConf = null;
+            try {
+                topoConf = readTopoConfAsNimbus(topologyId, topoCache);
+            } catch (KeyNotFoundException e) {
+                continue;
+            } catch (AuthorizationException e) {
+                continue;
+            } catch (IOException e) {
+                continue;
+            }
```

Comment:
I think it will be more compact to catch all on same line:
 catch (KeyNotFoundException | AuthorizationException | IOException e) {
       continue;
 }

Reply:
done